package.json

```diff
@@ -6,7 +6,8 @@
   "types": "framework/dist/local/index.d.ts",
```

Comment:
not sure if you wanted to do it in a separate MR, but we'll need to bump the version to do a release